release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -101,16 +101,47 @@ With this release, the installation program uses Cluster API instead of Terrafor
 [id="ocp-4-17-web-console_{context}"]
 === Web console
 
+[id="ocp-4-17-ocp-lightspeed_{context}"]
+==== {ols} is available in the web console
+
+Starting with {product-title} 4.16, {ols} is available for use in the web console.
+
+//https://github.com/openshift/openshift-docs/pull/81581
+//For more information, see xref:../web_console/capabilities_products-web-console.html#openshift-lightspeed-web-console_capabilities-web-console[Enhancing your experience in the {product-title} web console by using {ols-official}].
+
 [id="ocp-4-17-administrator-perspective_{context}"]
 ==== Administrator perspective
 
 This release introduces the following updates to the *Administrator* perspective of the web console:
 
+* Deprecated Operators are displayed in the OperatorHub before and after installation.
+* Config files for `MachineConfig` objects were moved to a new tab enabling you to check the content of the configuation files without having to manually retrieve its contents.
+* An alert was added to `operator-hub-item-details` and `operator-hub-subscribe` components if your cluster is on {gcp-first} with Workload Identity Foundation (WIF).
+* A page for ShipWright `BuildStrategy` was added with a `ClusterBuildStrategy` and `BuildStrategy` tab.
```

Comment:
Is this Dev Console new change? admin console doesn't have this new feature in 4.17

Reply:
I see it was added with this PR https://github.com/openshift/console/pull/14029

@sanketpathak could you weigh in on this?

Reply:
Yes this feature is added as part of epic [ODC-7518](https://issues.redhat.com/browse/ODC-7518)

Reply:
Will users know where to look for the new page?